src/app/View.ts

```diff
@@ -0,0 +1,115 @@
+import { Render } from './utils/Render';
+import { Timer } from './Timer';
+import { AnimalNames } from '../Enums/AnimalNamesEnum';
+import { Player } from '../Player';
+
+export class View {
+  renderMenuView(): void {
+    const inputName = Render.elementFactory('input', {
+      className: 'input__name',
+      placeholder: 'Carlos Santana',
+    });
+    const startGameButton = Render.elementFactory(
+      'button',
+      {
+        className: 'button__start button',
+      },
+      'Start Game',
+    );
+    const avatarsElements = Object.values(AnimalNames).map(
+      (animalName) => {
+        if (animalName === AnimalNames.BIG_DOG) {
+          return Render.elementFactory('img', {
+            src: `./static/images/avatars/dog.png`,
+          });
+        }
+        return Render.elementFactory('img', {
+          src: `./static/images/avatars/${animalName}.png`,
+        });
```

Comment:
here you create additional img with incorrect source (no 'small dog.png' file in this path)